pkg/estimator/service/doc.go

```diff
@@ -0,0 +1,4 @@
+package service
+
+//go:generate protoc --gogo_opt=paths=source_relative -I . -I ../../.. -I ../../../vendor --gogo_out=plugins=grpc:. --gogo_opt=Mpkg/estimator/pb/generated.proto=github.com/karmada-io/karmada/pkg/estimator/pb service.proto
+//go:generate mockery --name=EstimatorClient --inpackage
```

Comment:
We just generate the mock code for EstimatorClient, but have not used it yet, right?

Reply:
Right, not used yet until the descheduler is contributed.